java/test/src/main/java/org/ray/api/test/ResourcesManagementTest.java

```diff
@@ -37,7 +37,7 @@ public Integer echo(Integer number) {
     }
   }
 
-  @RayRemote(resources = {@ResourceItem(name = "CPU", value = 8),
+  @RayRemote(resources = {@ResourceItem(name = "CPU", value = 1000),
```

Comment:
why change this?

Reply:
In different machines, the number of CPUs is not determined, We should make sure that the machine doesn't have enough CPUs.

Reply:
then, we should set a FIXED number for this test.